src/components/hero2/icon-label.astro

```diff
@@ -0,0 +1,26 @@
+---
+const { icon, label } = Astro.props;
+
+const iconPath = `/icons/${icon}.svg`;
+---
+
+<!--<div class="flex items-center space-x-3">-->
+  <!-- Circle with SVG Icon -->
+  <!--<div class="w-10 h-10 bg-gray-200 rounded-full flex items-center justify-center">-->
+  <!--  <img src={iconPath} alt={label} class="w-6 h-6" />-->
+  <!--</div>-->
+
+   <div class="flex items-center space-x-3 px-6">
+        <div class="w-20 h-20 flex items-center justify-center rounded-full bg-[#D4D5E4]">
+          <img src={iconPath} alt={label} class="w-10 h-10" />
+        </div>
+        <!--<span class="text-lg font-medium text-[#7B7EB9]">Label</span>-->
+        <span class="text-xl font-medium text-[#7B7EB9]">{label}</span>
```

Comment:
The contrast ratio of this colour (used for the place and date) is a bit low and doesn't meet the WCAG accessibility guidelines:

<img width="641" alt="image" src="https://github.com/user-attachments/assets/d70fb8ce-d0ce-418d-b4fa-4d292f5df15a" />

<img width="235" alt="image" src="https://github.com/user-attachments/assets/49818ae9-586b-4759-b88d-870ed792e35b" />

The Chrome auto-suggestion is #616392 which is 4.6:

<img width="649" alt="image" src="https://github.com/user-attachments/assets/a55be809-37f1-4078-b614-9213c8e851e9" />


<img width="235" alt="image" src="https://github.com/user-attachments/assets/af73f6d7-9a4b-40cb-b85a-f60c222a288e" />

But it can be something else. Here's a contrast checker: https://webaim.org/resources/contrastchecker/

Reply:
@hugovk thanks for reporting! I've created an issue https://github.com/EuroPython/website/issues/1106